custom-ports/openblas/portfile.cmake

```diff
@@ -30,7 +30,7 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
         "dynamic-arch" DYNAMIC_ARCH
 )
 
-set(COMMON_OPTIONS -DBUILD_WITHOUT_LAPACK=ON)
+#set(COMMON_OPTIONS -DBUILD_WITHOUT_LAPACK=ON)
```

Comment:
It's a bummer the upstream port doesn't implement this as a feature flag, so that we can swap this setting without having to maintain our own custom port of it ☹️

Reply:
Agree. Looks like someone else wanted an option to have it on but it didn't go anywhere: https://github.com/microsoft/vcpkg/issues/34562